README.md

```diff
@@ -414,12 +443,16 @@ First load the `component_tags` tag library, then use the `component_[js/css]_de
     {% component_css_dependencies %}
 </head>
 <body>
-    {% component "calendar" date="2015-06-19" %}{% endcomponent %}
+    {% component "calendar" date="2015-06-19" / %}{% endcomponent %}
```

Comment:
Remove slash here, alternative is shown below.

Reply:
Fixed in https://github.com/EmilStenstrom/django-components/pull/572/commits/9061c3226ae70e2a23788063c3127d4136059752